repos/system_upgrade/el7toel8/actors/pcidevicesscanner/tests/test_pcidevicesscanner.py

```diff
@@ -0,0 +1,122 @@
+from leapp.snactor.fixture import current_actor_context, current_actor_libraries
```

Comment:
`current_actor_libraries` is deprecated and `current_actor_context` is injected under the hood. You can safely remove this line.
See example in https://leapp.readthedocs.io/en/latest/unit-testing.html#testing-private-actor-library-unit-tests